doc/source/whatsnew/v1.0.1.rst

```diff
@@ -0,0 +1,134 @@
+.. _whatsnew_101:
```

Comment:
we don't yet have a milestone or branch for this (will be 1.0.x after 1.0 release?), how do you get just this file to backport to a 1.0.1. I don't think this should be added yet

Reply:
Yea good point. Can do as separate PRs and only backport the one